src/utils/tests/inject-stylesheet.test.js

```diff
@@ -12,14 +12,39 @@ describe('injectStylesheet()', () => {
 	const env = '66c5421c67';
 	const uid = '97039403_1640028197170';
 
+	const triggerCallback = (mockedMutationsList) => {
+		Object.entries(callbacks).forEach(([id, callback]) => {
+			callback(mockedMutationsList);
+		});
+	};
+
+	let callbacks = {}
+
 	beforeEach(() => {
 		cleanup = jsdom();
+
+		const mutationObserverMock = class MU {
+			_id = Math.random();
+			callback = null;
+			constructor(callback) {
+				this.callback = callback;
+			}
+			observe = () => {
+				callbacks[this._id] = this.callback;
+			};
+			disconnect = () => {
+				delete callbacks[this._id];
+			};
+		};
+		window.MutationObserver = mutationObserverMock;
```

Comment:
Shouldn't this get reassigned to the original observer after the testing? It might be okay to leave it, but assigning global stuff like this always seems spicy

Reply:
Yeah I will. In this test, MutationObserver does not exist -- so there's no harm being done here